python/ray/experimental/workflow/serialization.py

```diff
@@ -0,0 +1,94 @@
+import asyncio
+from dataclasses import dataclass
+from enum import Enum
+import ray
+from ray.types import ObjectRef
+from ray.experimental.workflow.common import calculate_identifier
+from ray.experimental.workflow.storage import Storage
+from ray.experimental.workflow import workflow_storage
+from typing import Any, Dict, List, Optional, Tuple, TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from ray.actor import ActorHandle
+
+MANAGEMENT_ACTOR_NAME = "StorageManagementActor"
+MANAGEMENT_ACTOR_NAMESPACE = "workflow"
+
+
+def init_manager(storage: Storage) -> None:
+    handle = Manager.options(
+        name=MANAGEMENT_ACTOR_NAME,
+        namespace=MANAGEMENT_ACTOR_NAMESPACE,
+        lifetime="detached").remote(storage)
+    ray.get(handle.ping.remote())
+
+
+def get_manager() -> "ActorHandle":
+    return ray.get_actor(
+        MANAGEMENT_ACTOR_NAME, namespace=MANAGEMENT_ACTOR_NAMESPACE)
```

Comment:
Similar to workflow coordinator?

https://github.com/ray-project/ray/blob/master/python/ray/experimental/workflow/workflow_access.py#L391

In case of actor failure, it'll just start a new one.
Btw, also adding TODO there for better actor management.